### PR TITLE
test: add test for x-goog-api-client

### PR DESCRIPTION
**Pull request by bcoe**

### CODE REVIEW

File: src/project.ts [hunk 1]
```diff
@@ -262,7 +273,7 @@ class Project extends ServiceObject {
        */
       setMetadata: {
         reqOpts: {
-          method: 'PUT',
+          method: 'PUT' as METHOD_TYPE,
```
Comment (JustinBeckwith) [inline src/project.ts]: instead of duplicating the method list here, just type the `reqOpts` object as `Options`

Comment (callmehiphop) [inline src/project.ts]: Why do we need to cast this? Did this change in the common? I feel like if thats the case they should export the type.

File: test/headers.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            this.req.headers['x-goog-api-client'][0]
+          )
+        );
+        return {};
+      });
+    const [projects] = await resource.getProjects();
+    metadata.done();
+    req.done();
+  });
+  after(() => {
```
Comment (JustinBeckwith) [inline test/headers.ts]: I don't think the after here is strictly required 🤷‍♂

File: src/project.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /*!
- * Copyright 2015 Google Inc. All Rights Reserved.
+ * Copyright 2015 Google LLC
```
[the remark this answers is not in the document]
Comment (AVaksman) [reply src/project.ts]: I was once referred to this `autogen` regarding the licencing question.
https://github.com/mbrukman/autogen

**Comment by callmehiphop** [comment]
LGTM once lint task passes.

**Pull request merged**